docs/03_tutorials/21_tic-tac-toe-game-smart-contract-Testnet.md

```diff
@@ -4,10 +4,10 @@ link_text: "Tic-tac-toe Game Contract Using Testnet"
 ---
```

Comment:
Update changes to this file containing the EOSIO Testnet version if a change to the Single Node version also applies here.